testing/integration/test_ot.py

```diff
@@ -26,15 +32,46 @@ def test_ot_sequester(self):
         self._setup_stack()
         self.assertTrue(self._ping_host('forch-faux-1', '192.168.1.2'))
 
-        config = self._read_faucet_config()
-        interface = config['dps']['nz-kiwi-t2sw1']['interfaces'][1]
-        interface['native_vlan'] = 272
-        self._write_faucet_config(config)
+        self._generate_sequestering_config()
         time.sleep(5)
+
         self.assertTrue(self._ping_host('forch-faux-1', '192.168.2.1'))
         self.assertFalse(self._ping_host('forch-faux-1', '192.168.1.2'))
         self._clean_stack()
 
+    def _generate_sequestering_config(self):
```

Comment:
Is this actually an integration test or a unit test?  If it's a unit test, then it should be in a separate class FotUnitTests(UnitTestBase)  -- compared with class FotIntegrationTests(IntegrationTestBase):

Reply:
Added unit_base.py in python_lib and concrete unit test case in test_fot.py.